### ISSUE TITLE
xlsx imports should be more clear it is "Cadasta format"

### ISSUE THREAD

**Issue by wonderchook** [Cadasta/cadasta-platform#976]
I was able to fairly easily import data from Global Forest Watch using a csv format, when I tried to do the same with excel it didn't work. I think this is because I'm supposed to be reimporting "Cadasta" format data. Is that correct @bjohare?

As an enhancement we might want to let people import a flat file in xlsx the same as they do with CSV, but for the time being let's just try to make things clear to the user. 

[forest-read-import-excel.xlsx](https://github.com/Cadasta/cadasta-platform/files/635043/forest-read-import-excel.xlsx)
[forest-import.xlsx](https://github.com/Cadasta/cadasta-platform/files/635044/forest-import.xlsx)

**Comment by bjohare** [comment]
@wonderchook that's right, the excel import is just for 'Cadasta' format at the moment, ie it expects one or more of `locations`, `parties`, `relationships` worksheets to be present.

**Comment by wonderchook** [comment]
@clash99 @bethschechter thoughts on making this more clear for users? I was thinking with xlsx we might want to note "import data from another Cadasta project" or something like that.

**Comment by clash99** [comment]
@wonderchook as I work on the questionnaires in the upcoming weeks I will probably have more ideas...  for right now can we provide a sample file for them to download similar to what we do for questionnaires?  And then if they want to import it into the project they are able to do that.  Wouldn't these files have to be inline with the project questionnaire?  If so then we should have a sample for both the  minimal and standard questionnaires.

**Comment by clash99** [comment]
Related to #970 - we should be able eliminate the file type question (csv vs xlsx) based on what file type the user uploads.

<img width="832" alt="screenshot 2016-12-08 09 05 55" src="https://cloud.githubusercontent.com/assets/1119621/21014950/96ebaf8a-bd25-11e6-9549-d299d86afd1b.png">

**Comment by wonderchook** [comment]
I think eliminating that selection might make things a bit easier. I think we should wait until the user can import files in xlsx and csv that are the same format. Regarding the sample files to download it is true right now they need to define their schema. So maybe we just make a sample file of the minimum required fields and then have a link to the documentation that examples how to create a schema and additional fields?

**Comment by dpalomino** [comment]
@bethschechter 

This is to build something similar to the work that we did for xlsforms. Creating some sample CSV files showing how to import a basic project. 

I think for the XLSX format would not be necessary as it is using the downloading data from other Cadasta project.
